components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/internal/IdpMgtListenerServiceComponent.java

```diff
@@ -33,6 +33,9 @@
  */
 public class IdpMgtListenerServiceComponent {
 
+    private IdpMgtListenerServiceComponent(){
```

Comment:
Sonar suggested to introduce a private constructor to hide the defined public method.